CMakeLists.txt

```diff
@@ -16,7 +16,8 @@ include(${CMAKE_CURRENT_LIST_DIR}/cmake/utils.cmake)
 set(PYTHON_SUPPORTED_VERSIONS "3.8" "3.9" "3.10" "3.11")
 
 # Supported NVIDIA architectures.
-set(CUDA_SUPPORTED_ARCHS "7.0;7.5;8.0;8.6;8.9;9.0")
+set(CUDA_SUPPORTED_ARCHS "7.0;7.5;8.0;8.6;8.9;9.0;9.0a")
+set(CMAKE_CUDA_FLAGS "-gencode arch=compute_90a,code=sm_90a ${CMAKE_CUDA_FLAGS}")
```

Comment:
Why did you need to modify `CMAKE_CUDA_FLAGS`?  Does torch not detect 9.0a?

Reply:
No it doesn't unfortunately -- see https://github.com/pytorch/pytorch/commit/6e99f739235980e8d47e8fe6246c7466f2ce2f58 

I don't think this is the right fix, though so I am open to suggestions.